packages/chrome/src/styled/sheet/StyledSheet.ts

```diff
@@ -0,0 +1,81 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import styled, { ThemeProps, DefaultTheme } from 'styled-components';
+import { retrieveComponentStyles, DEFAULT_THEME } from '@zendeskgarden/react-theming';
+
+const COMPONENT_ID = 'chrome.sheet';
+
+interface IStyledSheetProps {
+  placement: 'start' | 'end';
+}
+
+const sheetSmartBorderStyle = ({
+  theme,
+  placement
+}: IStyledSheetProps & ThemeProps<DefaultTheme>) => {
+  let borderSide = '';
+
+  if (placement === 'end') borderSide = '-left';
+  else if (placement === 'start') borderSide = '-right';
+
+  return `border${borderSide}: 1px solid ${theme.palette.grey[300]};`;
+};
+
+export const StyledSheet = styled.aside.attrs({
+  'data-garden-id': COMPONENT_ID,
+  'data-garden-version': PACKAGE_VERSION,
+  tabIndex: -1
+})<IStyledSheetProps>`
+  display: flex;
+  position: relative;
+  flex-direction: column;
+  order: 1;
+
+  transition: width 0.25s ease-in-out;
+  will-change: width;
+
+  width: 380px;
+  min-height: 100%;
+
+  & > * {
```

Comment:
It looks like this selector applies a transition (`transition: opacity 0.5s ease-in-out)` to the children elements of the sheet. Is there a reason why header, body, footer, and the close button need that transition?

Reply:
It's there so when the container collapses or expands the contents from the sub-components don't appear squished.

Transition applied to the children:
![demo1](https://gyazo.com/a04436e3e47d0a8eb0ad5920a25f06c2.gif)

Without Transition applied to children:
![demo1](https://gyazo.com/524b3c255df5eed37c4a702b20a70415.gif)

Reply:
We've updated this and removed that selector.